### PR TITLE
Updating documentation to 4.0.0-SNAPSHOT

### PR DESCRIPTION
**Pull request by zsigmond-czine-everit**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -538,6 +538,22 @@ <h2 id="changing_bundles_without_restart">Changing bundles in the OSGi container
 </div>
 
       </div>
+<div>      
+  <h2 id="google_analytics_tracking">Google Analytics tracking</h2>
+  <p>
+    The eosgi-maven-plugin collect anonym usage statistics. The collected statistics use to obtain budget to further development.
```
Comment (attila-kiss-it) [inline index.html]: The eosgi-maven-plugin collects anonym usage statistics. The collected statistics are used to get budget fo further development.

File: index.html [hunk 2]
```diff
@@ -538,6 +538,22 @@ <h2 id="changing_bundles_without_restart">Changing bundles in the OSGi container
...
+    Collected data are:
+  </p>
+  <ul>
+    <li>analytics referer: use the eosgi-maven-plugin or other plugin. Example eclipse-e4-plugin, which is under development now.</li>
```
Comment (attila-kiss-it) [inline index.html]: analytics referer: the eosgi-maven-plugin or other plugin. Example eclipse-e4-plugin, which is under development now.

File: index.html [hunk 3]
```diff
@@ -538,6 +538,22 @@ <h2 id="changing_bundles_without_restart">Changing bundles in the OSGi container
...
+    <li>executed goal: the executed goal.</li>
+    <li>mac address hash: the MAC address hash to connect usage to anonym user. Use SHA-1 hash algorithm.</li>
```
Comment (attila-kiss-it) [inline index.html]: SHA-1 algorithm is used to hash the mac address.

File: index.html [hunk 4]
```diff
@@ -538,6 +538,22 @@ <h2 id="changing_bundles_without_restart">Changing bundles in the OSGi container
...
+    <li>plugin version: the plugin version.</li>
+  </ul>
+  <p>
+    If you want turn off to send usage statistic only add -Deosgi.analytics.skip=true property when build project.
```
Comment (attila-kiss-it) [inline index.html]: To turn off the Google Analytics tracking add the "-Deosgi.analytics.skip=true" property to the build comman or use the "<skipAnalytics>true</skipAnalytics>" plugin parameter.

**Comment by attila-kiss-it** [comment]
Generate the full maven site of the plugin and push the changes to the PR.

**Comment by attila-kiss-it** [comment]
Please update the documentation according to the current version 4.0.0-SNAPSHOT.

**Comment by zsigmond-czine-everit** [comment]
Updated documentation.

**Pull request merged**